notebooks/GP-Marginal.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<p>Have you tried with more informative priors? Like Exponential(1) or HalfNormal instead of HalfCauchy(5) for positive parameters? And more informative Gamma parameters?</p><p>I think these priors are quite wide, and the number of data points is quite low, which could explain why sampling is difficult</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
The priors don't have to be very informative, but it is helpful if they are bounded away from zero somewhat (and away from very large values).